tests/apis.js

```diff
@@ -1,88 +1,54 @@
+
 import apiUtils from '../lib/api/utils';
 import api from '../lib/api';
-import utils from '../lib/utils';
 import database from '../lib/database';
+import merged from './fixtures/live/objects/types-models-merged';
+import dbmocks from './fixtures/live/objects/database-mocks.js';
 
-const content = [];
-const types = [];
-const eachOfType = {};
-const allTypes = [];
-
-for (let i = 0; i < 5; i++) {
-  types.push(`Test Type ${ipsum({
-    count: 1,
-    units: 'words',
-  })}`);
-  eachOfType[types[i]] = 0;
-  allTypes.push({
-    name: types[i],
-    id: slugify(types[i]),
-    description: ipsum({
-      count: Math.round(Math.random() * 6 + 1),
-      units: 'words',
-    }),
-    attributes: [],
-  });
-}
+const content = dbmocks.rows;
+const types = merged.types;
+const eachOfType = dbmocks.eachOfType;
+const allTypes = merged.allTypes;
 
-for (let i = 0; i < 50; i++) {
-  const name = ipsum({
-    count: 3,
-    units: 'words',
-    format: 'plain',
+test.cb.before(t => {
+  database.init().then(() => {
+    database('live').insert(content).then(() => {
+      t.end();
+    }).catch(e => { // because we can't return in a before, this catch doesn't bubble out
+      t.fail(e);
+    });
+  }).catch(e => {
+    t.fail(e.message);
   });
+});
 
-  const sunrise = `${Math.round(Math.random() * 3 + 2016)}-${Math.round(Math.random() * 11 + 1)}-${Math.round(Math.random() * 25 + 1)}`;
-  const sunset = `${Math.round(Math.random() * 3 + 2016)}-${Math.round(Math.random() * 11 + 1)}-${Math.round(Math.random() * 25 + 1)}`;
-
-  const type = types[Math.round(Math.random() * 4)];
-
-  eachOfType[type] += 1;
-
-  const item = {
-    'id': uuid.v4(),
-    'language': 'en-us',
-    'sunrise': utils.time.iso(sunrise, '00:00'),
-    'sunrise-timezone': 'America/New_York',
-    'sunset': utils.time.iso(sunset, '00:00'),
-    'sunset-timezone': 'America/New_York',
-    'key': name,
-    'key-slug': slugify(name),
-    type,
-    'type-slug': slugify(type),
-    'attributes': {
-      name,
-      text: ipsum({
-        count: 3,
-        units: 'paragraphs',
-        format: 'html',
-      }),
-    },
-  };
-
-  content[i] = item;
-}
+//////////////////////////////
+// Utils - attributes
+//////////////////////////////
+test('Utils: attributes', t => {
+  const item = Math.round(Math.random() * content.length);
+  const expected = cloneDeep(content[item]);
+  const model = allTypes.find(typ => {
+    return typ.id === expected['type-slug'];
+  });
+  const attributes = apiUtils.attributes(expected.attributes, model.attributes);
 
-test.cb.before(t => {
-  Promise.map(content, item => {
-    return database('live').insert(item);
-  }).then(() => {
-    t.end();
-  }).catch(e => {
-    console.error(e);
-    t.end();
+  t.is(typeof attributes, 'object', 'Should return an object.');
+  Object.keys(attributes).map(key => {
+    const attr = key.split('-');
+    if (attr[attr.length - 1] === 'referencer') {
+      t.true(isUUID(attributes[key]), 'includes a uuid');
```

Comment:
So this test will be changing for https://github.com/punchcard-cms/punchcard/issues/408 right?